lib/cfndsl/aws/types.yaml

```diff
@@ -1074,6 +1075,9 @@ Types:
  EC2Tag:
   Key: String
   Value: String
+ ElastiCacheTag:
```

Comment:
Nope, EC2Tag works fine. I saw HostedZoneTag, Route53HealthCheckTags, etc and thought to follow that pattern, but I think it makes more sense to always use EC2Tag or rename them all to just Tag.